### DISCUSSION TITLE
CRAN release 1.0.0

### DISCUSSION THREAD

**Discussion by wlandau** [openpharma/brms.mmrm#111] [General]
{brms.mmrm} 1.0.0 is in really good shape. Together, we tackled custom marginal mean calculations and informative prior archetypes.

And at the end of last week, I updated our SBC checking workflow at https://openpharma.github.io/brms.mmrm/articles/sbc.html to include the different types of variance and correlation structures that are now possible. There are still open discussions at https://github.com/openpharma/brms.mmrm/discussions, but we have completed all the issues at https://github.com/openpharma/brms.mmrm/issues.

With all that in a nice state, I think it’s time for another CRAN release, and I am planning to submit it on June 4. @chstock, @yonicd, and @andrew-bean, do you have any concerns?

**Comment by chstock** [comment]
> With all that in a nice state, I think it’s time for another CRAN release, and I am planning to submit it on June 4. @chstock, @yonicd, and @andrew-bean, do you have any concerns?

I agree. No concerns. Great work!

**Reply by wlandau** [reply]
Thanks! Looks like we made it: https://cran.r-project.org/web/packages/brms.mmrm/
